sdk/storage/storage-file-share/review/storage-file-share.api.md

```diff
@@ -1071,6 +1089,7 @@ export interface LeaseOperationResponseHeaders {
     etag?: string;
     lastModified?: Date;
     leaseId?: string;
+    leaseTime?: number;
```

Comment:
I think I complained about this in the past but... leaseTime: number is a timestamp presumably?

Reply:
leaseTimeInSeconds

Reply:
leaseDuration ?

Reply:
leaseDuration is used for a string union type. Renamed to `leaseTimeInSeconds`.